src/App.jsx

```diff
@@ -0,0 +1,38 @@
+import React, {useState, useEffect, useMemo} from 'react';
+import ReactDOM from 'react-dom';
+import {HashRouter, Route, Switch, Link} from 'react-router-dom';
```

Comment:
I am still myself trying to understand best practices as it comes to JS and SEO. I know I have heard that HashRouter is not able to be easily read by Google. Here is a resource to help start exploring this more if you are interested: https://www.youtube.com/watch?v=rKgF0rf009c&list=PLKoqnv2vTMUPOalM1zuWDP9OQl851WMM9&index=7&t=0s

Reply:
Thanks dawnLunacy!  I ran out of time before getting a chance to write a default route in my express server, so I just swapped to HashRouter!  React-Router is definitely overkill here considering the content, but I wanted to do more until I realized 2.5hrs goes by REAL quick when you start with nothing.  I appreciate the review!